isos/bootstrap/bootstrap

```diff
@@ -41,7 +41,7 @@ if timeout --signal=KILL 2m mount -t ext4 /dev/disk/by-label/containerfs ${MOUNT
     # WARNING WARNING WARNING WARNING WARNING
     # if the tmpfs is not large enough odd hangs can occur and the ESX event log will
     # report the guest disabling the CPU
-    mount -t tmpfs -o size=64m tmpfs ${MOUNTPOINT}/.tether/
+    mount -t tmpfs -o size=80m tmpfs ${MOUNTPOINT}/.tether/
```

Comment:
Is there any way to put something into the logs so devs can at least see a warning message?

Reply:
A warning message about what? the warning from the comment above this change?

Reply:
We could echo a line but we cannot determine _which_ command will hang so we cannot target the logging to the point of failure. If we cannot rely on devs to at least glance over the code comments close to where the problem occurs then I doubt a not-proximate log message is going to help.
Having a build time script that _calculates_ the necessary footprint would be my preference, then we can right size it.